src/framework/theme/components/select/option-group.component.ts

```diff
@@ -17,7 +28,10 @@ import { convertToBoolProperty } from '../helpers';
     <ng-content select="nb-option, ng-container"></ng-content>
   `,
 })
-export class NbOptionGroupComponent {
+export class NbOptionGroupComponent implements AfterContentInit, OnDestroy {
+
+  protected alive: boolean = true;
```

Comment:
We don't need to specify `: boolean` explicitly here. Because ts compiler can infer type when you assign value in declaration. Also as it's quite obvious that this value always will be `boolean` 😄 
As I remember we have the following `tslint` rule.